### ISSUE TITLE
Adding other menus in main causes the searchbox to shrink

### ISSUE THREAD

**Issue by LunaticMuch** [h-enk/doks#160]
## Description

The search box has a decent size with the default menu list. If another voice is added, it gets squeezed to the right and become unusable/bad for the UI.

### Steps to reproduce

Add the following

```
[[main]]
  name = "MenuNew"
  url = "/something/"
  weight = 100

[[main]]
  name = "MenuOther"
  url = "/something/"
  weight = 200
```

### Expected result

The search box should maintain a size, a minimum one, until it collides with the menu.

### Actual result

Search box is squeezed and disappear.

**Comment by atwriter** [comment]
I just wanted to +1 this. The menu gets smaller and smaller as you add more [[main][] entries.

**Comment by h-enk** [comment]
Confirmed — will fix. Thx

**Comment by LunaticMuch** [comment]
@h-enk have you identified already the problem or just reproducted it?

**Comment by h-enk** [comment]
Both — will be fixed with https://github.com/h-enk/doks/issues/68

**Comment by nvolker** [comment]
Hey! Any news on this, also noticing it while piloting the theme.

**Comment by h-enk** [comment]
@nvolker, you can expect this to be available in 1-2 weeks.

**Comment by h-enk** [comment]
Will be looking something like this:

![Snag_25cb7cc](https://user-images.githubusercontent.com/3902872/129247148-f1349a29-b4b4-4ed8-946d-12a559a421f5.png)

See also: https://getdoks.org/docs/prologue/quick-start/

**Comment by h-enk** [comment]
Fixed w/ Doks [v0.3.4](https://github.com/h-enk/doks/releases/tag/v0.3.4)
